cristin-project/src/test/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandlerTest.java

```diff
@@ -253,42 +252,49 @@ void handlerReturnsBadRequestWhenTitleQueryParamIsEmpty() throws Exception {
         InputStream input = requestWithQueryParameters(Map.of(JsonPropertyNames.QUERY, EMPTY_STRING));
 
         handler.handleRequest(input, output, context);
-        var gatewayResponse = GatewayResponse.fromOutputStream(output,SearchResponse.class);
+        var gatewayResponse = GatewayResponse.fromOutputStream(output, SearchResponse.class);
 
         assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
         assertEquals(PROBLEM_JSON, gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
         assertThat(gatewayResponse.getBody(),
-                containsString(ErrorMessages.invalidQueryParametersMessage(JsonPropertyNames.QUERY,
-                        ErrorMessages.ALPHANUMERIC_CHARACTERS_DASH_COMMA_PERIOD_AND_WHITESPACE)));
+                   containsString(
+                       invalidQueryParametersMessage(
+                           JsonPropertyNames.QUERY,
+                           ALPHANUMERIC_CHARACTERS_DASH_COMMA_PERIOD_AND_WHITESPACE))
```

Comment:
now the error message might need a name and possibly value change since we now allow more types of characters in search string